src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'book',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books = state.books.concat(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.books = state.books.filter((book) => book !== action.payload);
+    },
+  },
+});
+
+module.exports = booksSlice.reducer;
+module.exports.booksActions = booksSlice.actions;
```

Comment:
- [ ] I would advise you to export the actions this way:

`export const { addBook, removeBook } = booksSlice.actions`. This will exposrt the book actions correctly. 👍 😎